llama_toolchain/agentic_system/client_sdk.py

```diff
@@ -0,0 +1,40 @@
+import fire
```

Comment:
@hardikjshah do you have any opinions on how we should situate these files?

Reply:
Another option is to put them alone with the SDK repos: https://github.com/stainless-sdks/llama-stack-python/tree/yanxi0830/dev

Reply:
yeah , the SDK repo's examples folder is probably best.

Reply:
Done (removed these scripts from the PR)! In the meantime (while SDK repo is getting ready), adding the `sdk_examples` to `llama-stack-apps` in https://github.com/meta-llama/llama-stack-apps/pull/66.